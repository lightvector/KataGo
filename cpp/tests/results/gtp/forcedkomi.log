: Running with following config:
allowResignation = true
cudaUseFP16 = false
cudaUseNHWC = false
forDeterministicTesting = true
hasButton = true
ignoreGTPAndForceKomi = 5.5
koRule = POSITIONAL
lagBuffer = 1.0
logAllGTPCommunication = true
logFile = tests/results/gtp/forcedkomi.log
logSearchInfo = true
logSearchInfoForChosenMove = false
logTimeStamp = false
logToStderr = false
maxPlayouts = 10000
maxTimePondering = 60.0
maxVisits = 100
multiStoneSuicideLegal = false
nnRandSeed = forTesting
nnRandomize = false
numSearchThreads = 1
openclUseFP16 = false
ponderingEnabled = false
resignConsecTurns = 3
resignThreshold = -0.90
rootSymmetryPruning = false
scoringRule = AREA
searchFactorAfterOnePass = 0.50
searchFactorAfterTwoPass = 0.25
searchFactorWhenWinning = 0.40
searchFactorWhenWinningThreshold = 0.95
searchRandSeed = forTesting
taxRule = NONE
trtUseFP16 = false

: GTP Engine starting...
: KataGo v1.16.4
: Using koPOSITIONALscoreAREAtaxNONEsui0button1 rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Initializing board with boardXSize 19 boardYSize 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: hasButton = true
: Config override: ignoreGTPAndForceKomi = 5.5
: Config override: koRule = POSITIONAL
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/forcedkomi.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: multiStoneSuicideLegal = false
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = false
: Config override: rules = 
: Config override: scoringRule = AREA
: Config override: searchRandSeed = forTesting
: Config override: taxRule = NONE
: Config override: trtUseFP16 = false
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: genmove_debug b
: MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . @ . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui0button1komi5.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 85
NN batches: 85
NN avg batch size: 1
PV: Q17 D3 C16 Q3 E16
Tree:
: T   4.67c W   4.33c S   0.47c ( +1.3 L  +1.3) N     100  --  Q17 D3 C16 Q3 E16
---Black(^)---
Q17 : T   5.46c W   4.93c S   0.54c ( +1.4 L  +1.3) LCB    2.28c P  8.97% WF  17.1 PSV      23 N      17  --  Q17 D3 C16 Q3 E16
Q3  : T   5.02c W   4.84c S   0.50c ( +1.4 L  +1.4) LCB    0.06c P  9.00% WF  13.0 PSV      13 N      13  --  Q3 D3 C16 Q17 E16
C16 : T   5.07c W   4.52c S   0.50c ( +1.4 L  +1.3) LCB    0.30c P  8.25% WF  12.0 PSV      12 N      12  --  C16 R16 D3 Q3 E16
R4  : T   4.77c W   4.63c S   0.38c ( +1.2 L  +1.2) LCB   -1.56c P  8.98% WF  11.0 PSV      11 N      11  --  R4 D3 C16 Q17 E16
R16 : T   4.92c W   4.39c S   0.53c ( +1.4 L  +1.3) LCB   -0.95c P  7.47% WF  11.0 PSV      11 N      11  --  R16 D3 C16 Q3 E16
C4  : T   4.44c W   4.34c S   0.49c ( +1.4 L  +1.3) LCB   -3.43c P  8.65% WF  10.0 PSV      10 N      10  --  C4 R16 C16 Q3 E16
D3  : T   4.23c W   4.39c S   0.48c ( +1.4 L  +1.4) LCB   -2.52c P  8.46% WF  11.0 PSV       9 N      11  --  D3 R16 C16 Q3 E16
D17 : T   4.21c W   3.58c S   0.41c ( +1.2 L  +1.2) LCB   -7.44c P  8.16% WF   8.0 PSV       8 N       8  --  D17 R16 D3 Q3 D5
Q4  : T   3.75c W   2.76c S   0.27c ( +1.0 L  +1.1) LCB  -39.48c P  5.54% WF   4.0 PSV       4 N       4  --  Q4 D3 D17
D16 : T   2.08c W   1.93c S   0.15c ( +0.9 L  +0.8) LCB -147.56c P  4.74% WF   2.0 PSV       2 N       2  --  D16 D3

: = R4
: Controller: komi 3.5
: = 
: Controller: genmove_debug w
: MoveNum: 1 HASH: F19857B2A2B00E3CA2C67ECEB0719FCA
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . @ . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . X1. .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui0button1komi5.5
Time taken: ###
Root visits: 100
New playouts: 89
NN rows: 77
NN batches: 77
NN avg batch size: 1
PV: D3 Q17 C16 Q15
Tree:
: T  -4.73c W  -4.44c S  -0.29c ( -1.2 L  -1.2) N     100  --  D3 Q17 C16 Q15
---White(^)---
D3  : T  -4.48c W  -4.26c S  -0.27c ( -1.2 L  -1.2) LCB   -7.05c P 14.87% WF  18.0 PSV      23 N      18  --  D3 Q17 C16 Q15
R16 : T  -4.42c W  -4.03c S  -0.20c ( -1.1 L  -1.2) LCB   -7.09c P 15.08% WF  19.0 PSV      19 N      19  --  R16 C4 C16 E4
Q17 : T  -4.29c W  -4.36c S  -0.21c ( -1.1 L  -1.1) LCB   -8.78c P 10.41% WF  13.0 PSV      13 N      13  --  Q17 C4 D17 E4
C4  : T  -4.61c W  -4.47c S  -0.35c ( -1.3 L  -1.3) LCB  -11.00c P  9.22% WF  11.0 PSV      11 N      11  --  C4 Q17 D17 Q15
C16 : T  -4.57c W  -4.49c S  -0.30c ( -1.2 L  -1.2) LCB  -11.20c P  8.07% WF  11.0 PSV      10 N      11  --  C16 Q17 D3 Q15
D17 : T  -5.13c W  -4.78c S  -0.31c ( -1.2 L  -1.3) LCB  -13.32c P  9.92% WF   9.0 PSV       9 N       9  --  D17 C4 Q17 E4
Q4  : T  -4.95c W  -4.16c S  -0.39c ( -1.4 L  -1.6) LCB  -24.85c P  5.55% WF   6.0 PSV       5 N       6  --  Q4 Q3 D3 C16 Q17 E16
Q16 : T  -6.08c W  -5.03c S  -0.40c ( -1.4 L  -1.4) LCB  -48.99c P  6.03% WF   4.0 PSV       4 N       4  --  Q16 D3 D17 D5
D4  : T  -5.97c W  -5.40c S  -0.53c ( -1.6 L  -1.6) LCB  -49.04c P  5.16% WF   4.0 PSV       4 N       4  --  D4 C16 Q17
D16 : T  -5.28c W  -4.79c S  -0.39c ( -1.4 L  -1.4) LCB  -48.20c P  4.64% WF   4.0 PSV       4 N       4  --  D16 D3 Q17 D5

: = Q17
: Controller: clear_board
: = 
: Controller: komi 4.5
: = 
: Controller: genmove_debug b
: MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . @ . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui0button1komi5.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 0
NN batches: 0
NN avg batch size: -nan
PV: Q17 D3 C16 Q3 E16
Tree:
: T   4.67c W   4.33c S   0.47c ( +1.3 L  +1.3) N     100  --  Q17 D3 C16 Q3 E16
---Black(^)---
Q17 : T   5.46c W   4.93c S   0.54c ( +1.4 L  +1.3) LCB    2.28c P  8.97% WF  17.1 PSV      23 N      17  --  Q17 D3 C16 Q3 E16
Q3  : T   5.02c W   4.84c S   0.50c ( +1.4 L  +1.4) LCB    0.06c P  9.00% WF  13.0 PSV      13 N      13  --  Q3 D3 C16 Q17 E16
C16 : T   5.07c W   4.52c S   0.50c ( +1.4 L  +1.3) LCB    0.30c P  8.25% WF  12.0 PSV      12 N      12  --  C16 R16 D3 Q3 E16
R4  : T   4.77c W   4.63c S   0.38c ( +1.2 L  +1.2) LCB   -1.56c P  8.98% WF  11.0 PSV      11 N      11  --  R4 D3 C16 Q17 E16
R16 : T   4.92c W   4.39c S   0.53c ( +1.4 L  +1.3) LCB   -0.95c P  7.47% WF  11.0 PSV      11 N      11  --  R16 D3 C16 Q3 E16
C4  : T   4.44c W   4.34c S   0.49c ( +1.4 L  +1.3) LCB   -3.43c P  8.65% WF  10.0 PSV      10 N      10  --  C4 R16 C16 Q3 E16
D3  : T   4.23c W   4.39c S   0.48c ( +1.4 L  +1.4) LCB   -2.52c P  8.46% WF  11.0 PSV       9 N      11  --  D3 R16 C16 Q3 E16
D17 : T   4.21c W   3.58c S   0.41c ( +1.2 L  +1.2) LCB   -7.44c P  8.16% WF   8.0 PSV       8 N       8  --  D17 R16 D3 Q3 D5
Q4  : T   3.75c W   2.76c S   0.27c ( +1.0 L  +1.1) LCB  -39.48c P  5.54% WF   4.0 PSV       4 N       4  --  Q4 D3 D17
D16 : T   2.08c W   1.93c S   0.15c ( +0.9 L  +0.8) LCB -147.56c P  4.74% WF   2.0 PSV       2 N       2  --  D16 D3

: = D3
: Controller: kata-set-rules chinese
: Changed rules to Chinese
: = 
: Controller: genmove_debug w
: MoveNum: 1 HASH: 399178EAA13E7B3665350D3C481B629B
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . X1. . . . . . . . . . . @ . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koSIMPLEscoreAREAtaxNONEsui0whbNfpok1komi5.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 76
NN batches: 76
NN avg batch size: 1
PV: Q3 R16 C16 D5
Tree:
: T  -4.03c W  -4.49c S   0.23c ( -1.3 L  -1.3) N     100  --  Q3 R16 C16 D5
---White(^)---
Q3  : T  -3.12c W  -3.95c S   0.34c ( -1.1 L  -1.1) LCB   -5.19c P 11.46% WF  21.1 PSV      33 N      21  --  Q3 R16 C16 D5
Q17 : T  -3.34c W  -4.16c S   0.32c ( -1.1 L  -1.2) LCB   -6.23c P 10.80% WF  17.0 PSV      17 N      17  --  Q17 Q3 C16 Q5
R16 : T  -4.09c W  -4.36c S   0.25c ( -1.2 L  -1.3) LCB   -7.68c P 13.44% WF  15.0 PSV      15 N      15  --  R16 Q3 C16 Q5
C16 : T  -3.52c W  -4.17c S   0.25c ( -1.3 L  -1.2) LCB   -7.39c P  9.04% WF  14.0 PSV      13 N      14  --  C16 R16 Q3 D5
R4  : T  -4.49c W  -5.16c S   0.12c ( -1.4 L  -1.4) LCB   -9.70c P 12.38% WF  12.0 PSV      12 N      12  --  R4 R16 D17 D5
D17 : T  -4.39c W  -4.97c S   0.19c ( -1.3 L  -1.3) LCB  -11.77c P 10.13% WF  10.0 PSV      10 N      10  --  D17 Q3 Q17 Q5
D4  : T  -5.94c W  -5.15c S   0.03c ( -1.6 L  -1.7) LCB  -25.90c P  9.16% WF   6.0 PSV       6 N       6  --  D4 C4 E3 C3 Q17 Q3
Q16 : T  -6.89c W  -5.03c S  -0.07c ( -1.7 L  -1.7) LCB -155.99c P  5.07% WF   2.0 PSV       2 N       2  --  Q16 C16
D16 : T  -6.59c W  -4.76c S  -0.02c ( -1.6 L  -1.6) LCB -155.69c P  4.94% WF   2.0 PSV       2 N       2  --  D16 R16

: = Q3
: GPU -1 finishing, processed 238 rows 238 batches
: All cleaned up, quitting
