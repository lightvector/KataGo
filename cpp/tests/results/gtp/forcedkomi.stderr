KataGo v1.8.0
Using koPOSITIONALscoreAREAtaxNONEsui0button1 rules initially, unless GTP/GUI overrides this
Loaded config configs/gtp_example.cfg and/or command-line and query overrides
Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
Model name: g170-b6c96-s175395328-d26788732
GTP ready, beginning main protocol loop
MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . @ . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui0button1komi5.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 95
NN batches: 95
NN avg batch size: 1
PV: Q17 D3 C16 Q3
Tree:
: T   4.60c W   3.54c S   1.06c ( +1.3 L  +1.3) N     100  --  Q17 D3 C16 Q3
---Black(^)---
Q17 : T   5.68c W   4.15c S   1.52c ( +1.5 L  +1.6) LCB   -2.75c P  8.97% WF  6.81% PSV      18 N      11  --  Q17 D3 C16 Q3
Q3  : T   4.81c W   4.05c S   0.76c ( +1.4 L  +1.5) LCB   -5.21c P  9.00% WF  6.72% PSV      10 N      10  --  Q3 R16 D3 D17
D3  : T   5.92c W   4.54c S   1.38c ( +1.5 L  +1.6) LCB   -4.36c P  8.45% WF  6.84% PSV      10 N      10  --  D3 R16 D17 Q3
R4  : T   4.63c W   3.89c S   0.74c ( +1.2 L  +1.4) LCB   -4.67c P  8.97% WF  6.70% PSV       9 N       9  --  R4 R16 D3 D17
C4  : T   4.27c W   3.13c S   1.14c ( +1.3 L  +1.4) LCB  -12.08c P  8.65% WF  6.66% PSV       9 N       9  --  C4 R16 D17 Q3
C16 : T   5.21c W   3.75c S   1.46c ( +1.5 L  +1.5) LCB   -9.57c P  8.25% WF  6.76% PSV       9 N       9  --  C16 D3 R16 Q3
D17 : T   4.21c W   2.96c S   1.25c ( +1.2 L  +1.3) LCB  -10.51c P  8.17% WF  6.65% PSV       9 N       9  --  D17 R16 D3 Q3
R16 : T   5.63c W   4.26c S   1.37c ( +1.6 L  +1.6) LCB   -5.90c P  7.46% WF  6.80% PSV       9 N       9  --  R16 D3 C16 Q3
Q4  : T   4.15c W   3.44c S   0.71c ( +1.0 L  +1.1) LCB  -25.49c P  5.55% WF  6.66% PSV       5 N       5  --  Q4 D3 D17
D16 : T   2.90c W   1.98c S   0.93c ( +0.9 L  +0.8) LCB -117.71c P  4.75% WF  6.55% PSV       4 N       4  --  D16 D3 Q17
MoveNum: 1 HASH: F19857B2A2B00E3CA2C67ECEB0719FCA
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . @ . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . X . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui0button1komi5.5
Time taken: ###
Root visits: 100
New playouts: 91
NN rows: 79
NN batches: 79
NN avg batch size: 1
PV: R16 D3 D17 D5
Tree:
: T  -4.90c W  -4.57c S  -0.33c ( -1.3 L  -1.4) N     100  --  R16 D3 D17 D5
---White(^)---
R16 : T  -4.17c W  -3.94c S  -0.23c ( -1.1 L  -1.2) LCB   -7.50c P 15.06% WF  9.26% PSV      23 N      18  --  R16 D3 D17 D5
D3  : T  -4.34c W  -4.14c S  -0.20c ( -1.2 L  -1.3) LCB   -7.62c P 14.85% WF  9.23% PSV      18 N      18  --  D3 Q17 D17 Q15
Q17 : T  -4.50c W  -4.47c S  -0.03c ( -1.1 L  -1.1) LCB   -8.56c P 10.42% WF  9.20% PSV      12 N      12  --  Q17 C4 D17 E4
D17 : T  -5.05c W  -5.03c S  -0.03c ( -1.4 L  -1.4) LCB  -11.59c P  9.93% WF  9.11% PSV      11 N      11  --  D17 Q17 D3 Q15
C4  : T  -5.26c W  -4.72c S  -0.53c ( -1.4 L  -1.6) LCB  -14.88c P  9.23% WF  9.09% PSV       9 N       9  --  C4 D17 Q17
C16 : T  -4.21c W  -4.48c S   0.27c ( -1.3 L  -1.5) LCB  -13.50c P  8.07% WF  9.24% PSV       9 N       9  --  C16 D3 Q17
Q16 : T  -6.16c W  -5.46c S  -0.70c ( -1.4 L  -1.5) LCB  -21.71c P  6.03% WF  8.98% PSV       6 N       6  --  Q16 D3 D17
Q4  : T  -5.35c W  -4.10c S  -1.25c ( -1.5 L  -1.8) LCB  -35.20c P  5.56% WF  9.08% PSV       6 N       6  --  Q4 Q3 D3 C16
D4  : T  -6.17c W  -5.40c S  -0.77c ( -1.6 L  -1.6) LCB  -78.71c P  5.16% WF  8.99% PSV       4 N       4  --  D4 C16 Q17
D16 : T  -6.48c W  -5.57c S  -0.91c ( -1.6 L  -1.6) LCB  -74.22c P  4.64% WF  8.95% PSV       4 N       4  --  D16 D3 Q17
MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . @ . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui0button1komi5.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 0
NN batches: 0
NN avg batch size: -nan
PV: Q17 D3 C16 Q3
Tree:
: T   4.60c W   3.54c S   1.06c ( +1.3 L  +1.3) N     100  --  Q17 D3 C16 Q3
---Black(^)---
Q17 : T   5.68c W   4.15c S   1.52c ( +1.5 L  +1.6) LCB   -2.75c P  8.97% WF  6.81% PSV      18 N      11  --  Q17 D3 C16 Q3
Q3  : T   4.81c W   4.05c S   0.76c ( +1.4 L  +1.5) LCB   -5.21c P  9.00% WF  6.72% PSV      10 N      10  --  Q3 R16 D3 D17
D3  : T   5.92c W   4.54c S   1.38c ( +1.5 L  +1.6) LCB   -4.36c P  8.45% WF  6.84% PSV      10 N      10  --  D3 R16 D17 Q3
R4  : T   4.63c W   3.89c S   0.74c ( +1.2 L  +1.4) LCB   -4.67c P  8.97% WF  6.70% PSV       9 N       9  --  R4 R16 D3 D17
C4  : T   4.27c W   3.13c S   1.14c ( +1.3 L  +1.4) LCB  -12.08c P  8.65% WF  6.66% PSV       9 N       9  --  C4 R16 D17 Q3
C16 : T   5.21c W   3.75c S   1.46c ( +1.5 L  +1.5) LCB   -9.57c P  8.25% WF  6.76% PSV       9 N       9  --  C16 D3 R16 Q3
D17 : T   4.21c W   2.96c S   1.25c ( +1.2 L  +1.3) LCB  -10.51c P  8.17% WF  6.65% PSV       9 N       9  --  D17 R16 D3 Q3
R16 : T   5.63c W   4.26c S   1.37c ( +1.6 L  +1.6) LCB   -5.90c P  7.46% WF  6.80% PSV       9 N       9  --  R16 D3 C16 Q3
Q4  : T   4.15c W   3.44c S   0.71c ( +1.0 L  +1.1) LCB  -25.49c P  5.55% WF  6.66% PSV       5 N       5  --  Q4 D3 D17
D16 : T   2.90c W   1.98c S   0.93c ( +0.9 L  +0.8) LCB -117.71c P  4.75% WF  6.55% PSV       4 N       4  --  D16 D3 Q17
Changed rules to Chinese
MoveNum: 1 HASH: 5CAAD553F3A4AF0F47459C51E40D103A
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . X . . . . . . . . . . . . . @ . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koSIMPLEscoreAREAtaxNONEsui0whbNkomi5.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 84
NN batches: 84
NN avg batch size: 1
PV: D3 Q17 Q3 Q15
Tree:
: T  -5.03c W  -5.15c S   0.12c ( -1.5 L  -1.5) N     100  --  D3 Q17 Q3 Q15
---White(^)---
D3  : T  -4.64c W  -4.96c S   0.32c ( -1.4 L  -1.4) LCB   -7.69c P 13.72% WF 10.10% PSV      21 N      16  --  D3 Q17 Q3 Q15
R16 : T  -4.33c W  -4.63c S   0.29c ( -1.3 L  -1.4) LCB   -7.77c P 13.35% WF 10.15% PSV      16 N      16  --  R16 Q3 C4 Q5
Q3  : T  -4.44c W  -4.85c S   0.41c ( -1.4 L  -1.5) LCB   -8.23c P 11.77% WF 10.13% PSV      14 N      14  --  Q3 C4 Q17 E4
R4  : T  -5.20c W  -5.47c S   0.27c ( -1.6 L  -1.6) LCB   -9.22c P 12.09% WF 10.00% PSV      13 N      13  --  R4 Q17 D3
Q17 : T  -4.64c W  -5.02c S   0.38c ( -1.4 L  -1.4) LCB   -8.90c P 11.10% WF 10.10% PSV      13 N      13  --  Q17 D3 Q3 E16
C4  : T  -5.49c W  -5.73c S   0.24c ( -1.7 L  -1.7) LCB  -17.40c P  8.26% WF  9.96% PSV       8 N       8  --  C4 R16 Q3
D16 : T  -6.25c W  -4.60c S  -1.65c ( -1.4 L  -1.6) LCB  -39.96c P  6.13% WF  9.86% PSV       6 N       6  --  D16 D17 E17 C17
Q16 : T  -5.88c W  -5.79c S  -0.08c ( -1.7 L  -1.7) LCB  -36.32c P  5.18% WF  9.92% PSV       5 N       5  --  Q16 D3 Q3
D4  : T  -6.07c W  -5.73c S  -0.34c ( -1.7 L  -1.6) LCB  -88.51c P  4.92% WF  9.90% PSV       4 N       4  --  D4 R16 Q3
Q4  : T  -6.16c W  -5.74c S  -0.42c ( -1.6 L  -1.6) LCB  -74.78c P  4.50% WF  9.89% PSV       4 N       4  --  Q4 D3 Q17
